src/__tests__/QueryParamProvider-test.tsx

```diff
@@ -1,8 +1,13 @@
 import * as React from 'react';
 import { render, cleanup } from '@testing-library/react';
-import { QueryParamProvider, QueryParamContext } from '../index';
+import { QueryParamProvider } from '../QueryParamProvider';
 import { makeMockLocation, makeMockHistory } from './helpers';
 
+// ¯\_(ツ)_/¯
```

Comment:
What should I do with these tests? I wanted to make sure the history-generating code in QueryParamProvider was still working, but I'm no longer passing history down via context.